### ISSUE TITLE
Placeholder doesn't appear when content is deleted

### ISSUE THREAD

**Issue by rlivsey** [bustlelabs/mobiledoc-kit#171]
[Post#isBlank](https://github.com/bustlelabs/content-kit-editor/blob/master/src/js/models/post.js#L17) returns false if you type something and then delete all the content resulting in the placeholder text not re-appearing.

The current implementation checks that there are no sections, but I'm guessing at this point there's one blank section.

Should this change to be blank if there are no sections OR one blank section?

Post#isBlank looks to be [used internally](https://github.com/bustlelabs/content-kit-editor/blob/f3feeee09d3e10974b8af2ce48c874283cf752a1/src/js/editor/editor.js#L583) in a few places, so not sure if that would break those.

Alternatively, there could be a public Editor#isBlank which is to be used instead & change the placeholder behaviour to use this?

**Comment by rlivsey** [comment]
Example mobiledoc from the demo before/after deleting all content:

```json
```

```json
{
  "version": "0.2.0",
  "sections": [
    [],
    [
      [
        1,
        "p",
        []
      ]
    ]
  ]
}
```

**Comment by bantic** [comment]
@rlivsey Good question. Some of the code surrounding the `post#isBlank` was to work around the difficulty of showing a placeholder when also showing a "cursorable" `<br>` (in order to make it so the cursor can be shown reliably, an empty markup section is rendered as a `<p><br></p>`). 

It may make the most sense to preserve the implementation of `post#isBlank` and instead ensure that when removing the last text from the only markup section it is also removed. That also allows the placeholder to reappear, which seems good. It also preserves a 1-1 mapping of a mobiledoc with a blank post, rather than having two visually-indistinguishable "blank" mobiledocs (your before/after mobiledocs, above).

What do you think?

**Comment by rlivsey** [comment]
@bantic yep that makes complete sense, I'll see if I can figure out how to make it do that…

**Comment by mixonic** [comment]
it is probably a little wonky, and will take some work on the renderer. I'd be happy to take a look next week.

**Comment by bantic** [comment]
closed by #461
